torch/csrc/jit/codegen/cuda/runtime/random_numbers.cu

```diff
@@ -87,3 +87,50 @@ __device__ float rng_uniform_rangef(
   auto uniform01 = rng_uniformf(rng_result, rng_component);
   return from + range * uniform01;
 }
+
+__device__ float normalf(unsigned int x, unsigned int y) {
+  float u = uniformf(x);
+  float v = uniformf(y) * 6.2831855f;
+
+  return sqrtf(-2.0f * logf(u)) * sinf(v);
+}
+
+__device__ double normal(unsigned int x0, unsigned int x1,
+                         unsigned int y0, unsigned int y1) {
+  double u = uniform(x0, x1);
+  double v = uniform(y0, y1) * 6.2831853071795860;
+
+  return sqrt(-2.0 * log(u)) * sin(v);
+}
+
+__device__ double rng_normal_standard(
+    const uint4& rng_result,
+    int rng_component) {
+  return normal(rng_result.x, rng_result.y, rng_result.z, rng_result.w);
+}
+
+__device__ float rng_normal_standardf(
+    const uint4& rng_result,
+    int rng_component) {
+  return normalf(
+    (&rng_result.x)[rng_component * 2],
+    (&rng_result.y)[rng_component * 2]);
+}
+
+__device__ double rng_normal_range(
+    const uint4& rng_result,
+    int rng_component,
+    double mean,
+    double std) {
```

Comment:
`torch.normal` uses mean and std instead of variance, so we also use std here.

https://pytorch.org/docs/stable/generated/torch.normal.html?highlight=normal#torch.normal